features/step_definitions/update-task.stepdefs.js

```diff
@@ -0,0 +1,121 @@
+const assert = require('assert');
+const { Given, When, Then } = require('@cucumber/cucumber');
+const TaskService = require('../../src/tasks/tasks.service');
+
+const taskService = new TaskService();
```

Comment:
You are setting up Task service right in the definitions, which is a not good approach for cucumber tests.

Step definitions should be abstract as possible, so they can be reused in any other app.

Better approach is to create world.js file, where app is initiated in BeforeAll handlers.
Then in tests you just trigger app endpoints. Refer to how test code is organised in our apps.

Refactor your code as well.

Reply:
Done